package.json

```diff
@@ -34,13 +34,13 @@
     "cordova-plugin-file": "6.0.1",
     "cordova-plugin-ionic-keyboard": "^2.1.3",
     "cordova-plugin-ionic-webview": "^4.2.1",
-    "cordova-plugin-scanbot-sdk": "4.8.0",
     "cordova-plugin-splashscreen": "^5.0.2",
     "cordova-plugin-statusbar": "^2.4.2",
     "cordova-plugin-whitelist": "^1.3.3",
     "cordova-sqlite-storage": "^5.1.0",
     "core-js": "^2.5.4",
     "rxjs": "~6.3.3",
+    "cordova-plugin-scanbot-sdk": "4.8.1-beta2",
```

Comment:
Can we change the version to 4.9.0 ? The last digit is for bugfixes, but ID-Card scanner is a new feature.

Reply:
@Nikituh Could we release the 4.9.0 beta so that I can update the plugin version?

Reply:
I don't really want to make another beta release just for the version number fix. I'll make the official release 4.9.0 and also update documentation